### DISCUSSION TITLE
Windows-IIS-Server

### DISCUSSION THREAD

**Discussion by jim55167** [nuxt/nuxt#15971] [Questions]
### Describe the feature

How deploy Nuxt 3 to Windows Server using IIS？
Currently documentation is non-existent for windows server deployments

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by skoenfaelt** [comment]
We are deploying in SSR and SSG. SSG is simple and for SSR you need that NodeJs IIS Module.
For SSR deployment, you need to build first, then copy .nuxt, static, nuxt.config, package.json (or directly the node_modules if its internal server without internet access)

Install node modules, if not copied.

Try this: https://stackoverflow.com/questions/46266609/host-node-js-on-windows-server-iis
https://github.com/nuxt/framework/discussions/9665#discussioncomment-4382178

> no need for expressJs

**Reply by prstwo** [reply]
Thank you. that link really helped, but how about **web.config**? what it should be written for nuxt?

**Comment by brunotourinho** [comment]
Hello guys, I would love a more detailed solution to this issue. i.e.: how to deploy to an existing website, deploys in a subfolder. It would be great if you can share some code, to help me clarify!

**Comment by skoenfaelt** [comment]
https://github.com/nuxt/nuxt/issues/20499
https://github.com/nuxt/framework/commit/ede534847831f46c45089a122d19b083d2ae26c7

please have a look. We tried with Nuxt3 [+Docus] and worked 

best on Nuxt3 / Nitro is, you don´t have to copy the whole node_modules folder, since Nitro bundle it in the .outputs folder
- from 600mb node_modules to 30mb 

Those Nuxters are incredible!
